packages/worker/src/worker.ts

```diff
@@ -441,11 +440,11 @@ export class Worker {
   }
 
   protected static async createWorkflowCreator(
-    bundle: WorkflowBundleWithSourceMap,
+    bundle: WorkflowBundleAndCodePath,
     compiledOptions: CompiledWorkerOptions
   ): Promise<WorkflowCreator> {
     if (compiledOptions.debugMode) {
-      return await VMWorkflowCreator.create(bundle.code, bundle.sourceMap, compiledOptions.isolateExecutionTimeoutMs);
+      return await VMWorkflowCreator.create(bundle.code, bundle.codePath, compiledOptions.isolateExecutionTimeoutMs);
```

Comment:
Wouldn't the path be encoded in the bundle's source map?

Reply:
Same